README.rst

```diff
@@ -70,7 +70,8 @@ The janitor is configured via command line args, environment variables, Kubernet
 Supported Kubernetes annotations:
 
 ``janitor/ttl``
-    Maximum time to live (TTL) for the annotated resource. Annotation value must be a string composed of a integer value and a unit suffix (one of ``s``, ``m``, ``h``, ``d``, or ``w``), e.g. ``120s`` (120 seconds), ``5m`` (5 minutes), ``8h`` (8 hours), ``7d`` (7 days), or ``2w`` (2 weeks).
+    Minimum time to live (TTL) for the annotated resource. Annotation value must be a string composed of a integer value and a unit suffix (one of ``s``, ``m``, ``h``, ``d``, or ``w``), e.g. ``120s`` (120 seconds), ``5m`` (5 minutes), ``8h`` (8 hours), ``7d`` (7 days), or ``2w`` (2 weeks).
```

Comment:
The change here is `Maximum` -> `Minimum`

Reply:
Why "Minimum"?

Reply:
Because it is minimum time to live, not maximum time to live. 
Maximum time to live means that instance can die on a half way to the value specified (at least I thought like that).
I can change it back - later it is described precisely.

Reply:
I think "Maximum" is more correct as anybody could delete the resource before the TTL expires --- kube-janitor just ensures that it lives no longer (+/- minutes) than the defined TTL.

Reply:
Fixed https://github.com/hjacobs/kube-janitor/pull/39/commits/2b399836993aa1a33a925566c0eb70cdccd86c1b